CONTRIBUTORS.md

```diff
@@ -1,14 +1,18 @@
+# Contributors
+
 WordPress.org Contributor Usernames
```

Comment:
Updating the above would be great, probably remove this line altogether and change the title on line 1 to `Contributor Usernames`

Reply:
:+1: